packages/itmat-commons/src/models/study.ts

```diff
@@ -27,7 +29,7 @@ export interface IRole {
     studyId: string;
     name: string;
     permissions: string[];
-    users: string[];
+    users: Models.UserModels.IUser[];
```

Comment:
Already talked about this

Reply:
Yes, but IUser is still the reflection of that gets returned by the database. There is a larger work to be done to enable more accurate type inference. IRole is also impacted by this. We would need to really make sure this gets thoroughly looked at later.

Reply:
no but IRole users in mongodb is string[]

Reply:
it's definitely not IUser[] in mongo